hammer/synthesis/genus/__init__.py

```diff
@@ -322,21 +339,35 @@ def syn_generic(self) -> bool:
             if len(inverter_cells) > 0 and len(logic_cells) > 0:
                 # Clock mapping needs at least the attributes cts_inverter_cells and cts_logic_cells to be set
                 self.append("set_db map_clock_tree true")
-        self.verbose_append("syn_generic")
+
+        if self.get_setting("synthesis.genus.phys_flow_effort").lower() == "none":
+            self.verbose_append("syn_generic")
+        else:
+            self.verbose_append(f"read_def {self.get_setting('synthesis.genus.def_file')}")
+            self.verbose_append("syn_generic -physical")
 
         self.dedup_ilms()
 
         return True
 
     def syn_map(self) -> bool:
-        self.verbose_append("syn_map")
+        if self.get_setting("synthesis.genus.phys_flow_effort").lower() == "none":
+            self.verbose_append("syn_map")
+        else:
+            self.verbose_append("syn_map -physical")
         # Need to suffix modules for hierarchical simulation if not top
         if self.hierarchical_mode not in [HierarchicalMode.Flat, HierarchicalMode.Top]:
             self.verbose_append("update_names -module -log hier_updated_names.log -suffix _{MODULE}".format(MODULE=self.top_module))
 
         self.dedup_ilms()
 
         return True
+    
+    def ispatial_opt(self) -> bool:
```

Comment:
I would rather move this if statement into `syn_map`, since the entire step is conditional.